### ISSUE TITLE
Regression: Drag&drop a note from the list into the (basic) editor no longer inserts a link.

### ISSUE THREAD

**Issue by jtikalsky** [laurent22/joplin#8734]
In the latest version, I find that I am no longer able to drag&drop a note from the list of notes into the plaintext editor to create a link to that note.  I have used this feature a lot in the past, and still do on some less up-to-date machines.

## Environment

```
Joplin 2.11.11 (prod, win32)

Client ID: [...]
Sync Version: 3
Profile Version: 43
Keychain Supported: Yes

Revision: 6886f6f
```

(Windows 11)

## Steps to reproduce

1. Open note (basic editor)
2. Drag another note from the list of notes on the left into the text editor.

## Expected/past outcome

A markdown link is dropped into the editor.

## Actual/current outcome

A `⊘` icon is shown on the cursor, and the drop fails.

**Comment by personalizedrefrigerator** [comment]
This should be fixed in the version 2.12 pre-release (see https://github.com/laurent22/joplin/issues/8370).

(If it isn't fixed, please let us know!)

**Comment by jtikalsky** [comment]
:+1:

Thanks.  I'll test it out this evening.
